verification/cbmc/sources/make_common_data_structures.c

```diff
@@ -178,6 +178,8 @@ struct aws_cryptosdk_hdr *ensure_nondet_hdr_has_allocated_members(const size_t m
     struct aws_cryptosdk_hdr *hdr = malloc(sizeof(*hdr));
     if (hdr != NULL) {
         hdr->alloc = nondet_bool() ? NULL : can_fail_allocator();
+        enum aws_cryptosdk_alg_props alg_id;
+        hdr->alg_id = alg_id;
```

Comment:
Why do you need that here? If `alg_id` is a enum, than CBMC will assign a non-det value for it by default.

Reply:
Gotcha, I've removed this now. You're right: it isn't needed.